src/pages/admin/ReportDashboard.tsx

```diff
@@ -0,0 +1,156 @@
+import { useEffect, useState } from 'react';
+import {
+  Input,
+  Select,
+  Spinner,
+  Text,
+  VStack,
+  Box,
+  Heading,
+  Collapse,
+  Button,
+  useToast,
+  HStack,
+} from '@chakra-ui/react';
+import { getAllReport } from '../../services/report';
+import { Report, ReportType } from '../../types';
+import { devPrint } from '../../components/utils/RandomUtils';
+
+const ReportDashboard = () => {
+  const [reports, setReports] = useState<Report[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [searchKeyword, setSearchKeyword] = useState('');
+  const [filterType, setFilterType] = useState<ReportType | ''>('');
+  const [filterUserId, setFilterUserId] = useState<number | ''>('');
+  const [expandedReportId, setExpandedReportId] = useState<string | null>(null);
```

Comment:
Can leave blank instead of `null` for `string | undefined`

e.g.
```ts
const [expandedReportId, setExpandedReportId] = useState<string | null>(null);
```

becomes

```ts
const [expandedReportId, setExpandedReportId] = useState<string>();
```